frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -117,13 +120,25 @@ export const MainLayout = forwardRef<HTMLDivElement, IMainLayoutProps>(
                             )}
                             show={<DraftBanner project={projectId || ''} />}
                         />
-                        <StyledMainLayoutContent item xs={12} sm={12} my={2}>
-                            <MainLayoutContentContainer ref={ref}>
-                                <BreadcrumbNav />
-                                <Proclamation toast={uiConfig.toast} />
-                                {children}
-                            </MainLayoutContentContainer>
-                        </StyledMainLayoutContent>
+                        <Box sx={{ display: 'flex', mt: '2px' }}>
```

Comment:
We need to wrap here so that flex applies only to sidebar and main layout and excludes draft CR banner from the flex positioning

Reply:
Should use theme.spacing(0.25)

Reply:
will update in subsequent PR